scrc/dataset_creation/report_creator.py

```diff
@@ -270,17 +274,18 @@ def report_references_not_found(self, not_found_list, label):
             for item in updated_list:
                 f.write(f"{item}\n")
         # report
-        df = pd.DataFrame({'chamber': [], 'year': []})
+        df = pd.DataFrame({'chamber': [], 'year': [], 'legal_area': []})
         for item in updated_list:
             chamber = item.split('_', 2)[0]
+            legal_area = get_legal_area_bger(chamber)
             # TODO get nice representation of chamber
             chamber = chamber[0]
             year = int(item.split('/', 2)[1])
-            s_row = pd.Series([chamber, year], index=df.columns)
+            s_row = pd.Series([chamber, year, legal_area], index=df.columns)
             df = df.append(s_row, ignore_index=True)
         df = df[df['year'] > 2000]
         df = df[df['year'] < 2030]
-        self.plot_attribute_color(df, 'year', 'chamber', 'references_not_found')
+        self.plot_attribute_color(df, 'year', 'legal_area', 'references_not_found')
 
```

Comment:
please name it consistently (legal area or law area).

Reply:
changed it to law